ravenframework/JobHandler.py

```diff
@@ -105,6 +105,25 @@ def __init__(self):
     self.rayInstanciatedOutside = None
     self.remoteServers = None
 
+  def __getstate__(self):
+    """
+      This function return the state of the JobHandler
+      @ In, None
+      @ Out, state, dict, it contains all the information needed by the ROM to be initialized
+    """
+    state = copy.copy(self.__dict__)
+    state.pop('_JobHandler__queueLock')
```

Comment:
Nono it should be ``_JobHandler__queueLock``. The reason is that since ```__queueLock``` is a "private" variable, in the dict is stored as ``_**ClassName**__varName``

Reply:
```
>>> class foo():
...   def __init__(self):
...     self.__queueLock = "myPrivateVariable"
... 
>>> myfoo = foo()
>>> myfoo.__dict__
{'_foo__queueLock': 'myPrivateVariable'}
```